File: BmiCalculator/app/src/main/java/com/example/bmicalculator/MainActivity.java
package com.example.bmicalculator;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity
{
    private Button startButton;
    private Button quizButton;
    private Button covidInfoButton;

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        covidInfoButton = (Button) findViewById(R.id.covidInfoButton);
        quizButton = (Button) findViewById(R.id.quizButton);
        startButton = (Button) findViewById(R.id.StartButton);

        covidInfoButton.setOnClickListener(new View.OnClickListener()
        {
            @Override
            public void onClick(View v) { covidInfoActivity(); }
        });

        quizButton.setOnClickListener((new View.OnClickListener()
        {
            @Override
            public void onClick(View v) { quizActivity(); }
        }));

        startButton.setOnClickListener(new View.OnClickListener()
        {
            @Override
            public void onClick(View v)
            {
                calculatorActivity();
            }
        });
    }

    public void quizActivity()
    {
        Intent intent = new Intent(this, QuizActivity.class);
        startActivity(intent);
    }

    public void covidInfoActivity()
    {
        Intent intent = new Intent(this, CovidInfoActivity.class);
        startActivity(intent);
    }

    public void calculatorActivity()
    {
        Intent intent = new Intent(this,CalculatorActivity.class);
        startActivity(intent);
    }
}

File: BmiCalculator/app/src/main/java/com/example/bmicalculator/MainActivity.kt
package com.example.bmicalculator

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private var startButton: Button? = null
    private var quizButton: Button? = null
    private var covidInfoButton: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        covidInfoButton = findViewById<View>(R.id.covidInfoButton) as Button
        quizButton = findViewById<View>(R.id.quizButton) as Button
        startButton = findViewById<View>(R.id.StartButton) as Button
        covidInfoButton!!.setOnClickListener { covidInfoActivity() }
        quizButton!!.setOnClickListener { quizActivity() }
        startButton!!.setOnClickListener { calculatorActivity() }
    }

    fun quizActivity() {
        val intent = Intent(this, QuizActivity::class.java)
        startActivity(intent)
    }

    fun covidInfoActivity() {
        val intent = Intent(this, CovidInfoActivity::class.java)
        startActivity(intent)
    }

    fun calculatorActivity() {
        val intent = Intent(this, CalculatorActivity::class.java)
        startActivity(intent)
    }
}